app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -243,18 +243,18 @@ class ConversationViewModel @Inject constructor(
             val sender = members.findSender(message.senderUserId)
             MessageViewWrapper(
                 messageContent = fromMessageModelToMessageContent(message),
-                messageSource = MessageSource.CurrentUser,
+                messageSource = if(sender is MemberDetails.Self) MessageSource.Self else MessageSource.OtherUser,
                 messageHeader = MessageHeader(
                     // TODO: Designs for deleted users?
-                    username = sender?.name ?: "Deleted User",
+                    username = sender.name ?: "Deleted User",
```

Comment:
Since we are changing it, can we extract the “Deleted User” string to const ? To easily change if Designs changed it